src/chroot_script

```diff
@@ -212,6 +212,25 @@ echo "mapping wlan1" >> /etc/network/interfaces
 echo "  script /root/bin/map_iface" >> /etc/network/interfaces
 echo "source /boot/octopi-network.txt" >> /etc/network/interfaces
 
+# allow wifi network configuration via /boot/octopi-wifi.txt
+mv /etc/wpa_supplicant/wpa_supplicant.conf /boot/octopi-wifi.txt
+ln -s /boot/octopi-wifi.txt /etc/wpa_supplicant/wpa_supplicant.conf
```

Comment:
Thank you so much @foosel for pointing to the blog post about wpa_supplicant.conf ! I will test it out tonight. It may just work and my PR will be totally unnecessary.